extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -175,8 +175,17 @@ public boolean isAnyPropertySet() {
         @ConfigItem
         public Optional<Integer> statementBatchSize;
 
+        /**
+         * Whether Hibernate should quote all identifiers.
+         * <p>
+         * Default value is false.
+         */
+        @ConfigItem
+        public Optional<Boolean> globallyQuotedIdentifiers;
```

Comment:
I'm not convinced we should add this one to the `jdbc` section. It has something to do with the schema generation so I would add it to the `database` section.